k3po-maven-plugin/src/main/java/org/kaazing/k3po/maven/plugin/internal/AbstractMojo.java

```diff
@@ -28,23 +29,19 @@
 
     private static final ThreadLocal<RobotServer> ROBOT_SERVER = new ThreadLocal<>();
 
-    /**
-     * @parameter expression="${project}"
-     * @required
-     * @readonly
-     * @since 1.0
-     */
+    @Parameter(defaultValue = "${project}", readonly = true)
     protected MavenProject project;
 
-    /**
-     * @parameter default-value="false" expression="${skipTests || skipITs}"
-     */
+    @Parameter(defaultValue = "false", property = "skipTests")
```

Comment:
I think this change is consistent with the previous semantics, just split into 2 parameters as part of updating to use the Maven Java annotations, with the "or" logic moved to the goal implementation instead of the javadoc comment.

Regarding the logic itself, the `failsafe plugin` appears to support the skipping of tests via the `skipTests` property, but given the effect of `skipTests` to cause both unit tests and integration tests to be skipped, another more targeted `skipITs` is provided to have the same effect on the `failsafe plugin` (skipping ITs) but not also causing other plugins to skip unit tests, such as the `surefire plugin`.

In our case, the `k3po plugin` also supports both `skipTests` and `skipITs` for the same reasons, either use `skipTests` to skip all unit tests and integration tests, or use `skipITs` to skip only integration tests. In either case, the `k3po plugin` would skip setting up the server since the `failsafe plugin` would not be executing the integration tests.